src/connection/connection_support.c

```diff
@@ -2945,3 +2957,213 @@ css_conn_entry::init_pending_request ()
+    }
+  else
+    {
+      // Linux and Unix
+      connection_protocol = SERVER_REQUEST;
+    }
+
+  if (css_common_connect
+      (conn, &rid, hname, connection_protocol, message_to_master, message_to_master_length, master_port_id, 0,
+       true) == NULL)
+    {
+      goto fail_end;
+
+    }
+
+  if (css_readn (conn->fd, (char *) &response_buff, sizeof (int), -1) != sizeof (int))
+    {
+      goto fail_end;
+    }
+
+  response = ntohl (response_buff);
+
+  TRACE ("css_connect_to_master_server received %d as response from master\n", response);
+
+  switch (response)
+    {
+    case SERVER_ALREADY_EXISTS:
+#if !defined (SERVER_MODE)
+      if (IS_MASTER_CONN_NAME_HA_COPYLOG (message_to_master))
+	{
+	  er_set (ER_ERROR_SEVERITY, ARG_FILE_LINE, ERR_CSS_COPYLOG_ALREADY_EXISTS, 1,
+		  GET_REAL_MASTER_CONN_NAME (message_to_master));
+	}
+      else if (IS_MASTER_CONN_NAME_HA_APPLYLOG (message_to_master))
+	{
+	  er_set (ER_ERROR_SEVERITY, ARG_FILE_LINE, ERR_CSS_APPLYLOG_ALREADY_EXISTS, 1,
+		  GET_REAL_MASTER_CONN_NAME (message_to_master));
+	}
+      else if (IS_MASTER_CONN_NAME_HA_SERVER (message_to_master))
+	{
+	  er_set (ER_ERROR_SEVERITY, ARG_FILE_LINE, ERR_CSS_SERVER_ALREADY_EXISTS, 1,
+		  GET_REAL_MASTER_CONN_NAME (message_to_master));
+	}
+      else
+	{
+	  er_set (ER_ERROR_SEVERITY, ARG_FILE_LINE, ERR_CSS_SERVER_ALREADY_EXISTS, 1, message_to_master);
+	}
+#else // SERVER_MODE
+      er_set (ER_ERROR_SEVERITY, ARG_FILE_LINE, ERR_CSS_SERVER_ALREADY_EXISTS, 1, message_to_master);
+#endif // SERVER_MODE
+      goto fail_end;
+
+    case SERVER_REQUEST_ACCEPTED_NEW:
+      /*
+       * Master requests a new-style connect, must go get
+       * our port id and set up our connection socket.
+       * For drivers, we don't need a connection socket and we
+       * don't want to allocate a bunch of them.  Let a flag variable
+       * control whether or not we actually create one of these.
+       */
+      if (css_Server_inhibit_connection_socket)
+	{
+	  server_port_id = -1;
+	}
+      else
+	{
+	  server_port_id = css_open_server_connection_socket ();
+	}
+      response = htonl (server_port_id);
+      css_net_send (conn, (char *) &response, sizeof (int), -1);
+      /* this connection remains our only contact with the master */
+      return conn;
+
+    case SERVER_REQUEST_ACCEPTED:
+#if defined(WINDOWS)
+      /* Windows can't handle this style of connection at all */
+      er_set (ER_ERROR_SEVERITY, ARG_FILE_LINE, ERR_CSS_ERROR_DURING_SERVER_CONNECT, 1, message_to_master);
+      goto fail_end;
+#else /* WINDOWS */
+      /* send the "pathname" for the datagram */
+      /* be sure to open the datagram first.  */
+      pname = std::filesystem::temp_directory_path ();
+
+#if !defined (SERVER_MODE)
+      pname += "/csql_tcp_setup_server" + std::to_string (getpid ());
+#else // SERVER_MODE
+      pname += "/cubrid_tcp_setup_server" + std::to_string (getpid ());
+#endif // SERVER_MODE
```

Comment:
More accurate names for these files would be "client_master" and "server_master"
```suggestion
#if !defined (SERVER_MODE)
      pname += "/client_master_tcp_setup_server" + std::to_string (getpid ());
#else // SERVER_MODE
      pname += "/server_master_tcp_setup_server" + std::to_string (getpid ());
#endif // SERVER_MODE
```